sfge/src/test/java/com/salesforce/graph/ops/MethodUtilTest.java

```diff
@@ -334,136 +329,4 @@ public void getTargetMethods_targetMethodInInnerAndOuterClass() {
                 messages,
                 containsString(EventKey.WARNING_MULTIPLE_METHOD_TARGET_MATCHES.getMessageKey()));
     }
-
-    @ValueSource(
-            strings = {
-                Schema.AURA_ENABLED,
-                Schema.INVOCABLE_METHOD,
-                Schema.REMOTE_ACTION,
-                Schema.NAMESPACE_ACCESSIBLE
-            })
-    @ParameterizedTest(name = "{displayName}: {0}")
-    public void testGetMethodsWithAnnotation(String annotation) {
```

Comment:
Could you point me to where these tests now live?

Reply:
Line 325 of `PathEntryPointUtilTest.java`.